src/stock_str.rs

```diff
@@ -614,6 +620,29 @@ pub(crate) async fn secure_join_replies(context: &Context, contact_id: u32) -> S
     }
 }
 
+/// Stock string: `Scan to chat with %1$s`.
+pub(crate) async fn verify_contact_qr_description(
```

Comment:
we are not speaking about "verify contact" and the protocol is called "setup contact", so maybe better name the function `verify_contact_qr_description`